docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+#### Verifiable Credential Proofs
+
+In addition to the normal signature, a `MerkleKeySignature2021` requires additional proof data to validate the signature's origin and validaty. The public key itself needs to be revealed inside the `signatureValue` field, but also the individual hashes inside the Merkle Tree and their relative location in the path to create a valid `Proof of Inclusion`. 
+
+#### Revocation List
+
+In order to revoke a public key, and therefore any Verifiable Credential or other statements, the DID Document must be updated with a revocation list. The REQUIRED `revocation` field is introduced inside the `MerkleKeyCollection2021` verification method, which lists the indices from the leaves of the Merkle Tree that are revoked. These indices are further compressed via [Roaring Bitmaps](https://roaringbitmap.org/).
+
+### Standardized Services
+
+The IOTA Identity framework also standardized certain `services` that are embedded in the DID Document. It is RECOMMENDED to implement these when implementing the `did:iota` method. 
+
+Currently standardized `services`:
+* Nothing yet. 
+
+## Security Considerations
+
+The `did:iota` method is implemented on the IOTA Tangle, a public permissionless and feeless Distributed Ledger Technology (DLT), making it resistant against almost all censorship attack vectors. Up until the `Coordicide` update for the IOTA network, a reliability on the coordinator exists for resolving ordering conflicts. This has a minor censorship possibility, that, in the worst case, can prevent ordering conflicts from being resolving in a DID. However, these can only be published by the possessor of the private key and therefore does not provide a usable attack vector. Lastly, a node may decide to censor DID messages locally, however a user can easily use another node. 
+
+Since DID messages are always to be signed and the 'chain of custody' of the signing key can be traced throughout the identity lifespan, replay, message insertion, deletion, modification and man-in-the-middle attacks are prevented.
+
+### Stateless Identities
+
+Unlike for-purpose Blockchains or Smart contract based DID methods, the IOTA tangle does not track and store the state of DID Document. This prevents dusting attacks against the nodes. However, the client that resolves the DID Document has the responsibility to always validate the entire history of the DID Document as to validate the 'chain of custody' of the signing keys. If a client does not do this, identity hijacking is easily possible. The IOTA Identity framework, which implements the `did:iota` method, provides an easy to use client-side validation implementation. 
```

Comment:
Blockchains -> blockchains ? Dunno.
Smart contract -> Smart Contract
IOTA tangle -> IOTA Tangle 
is easily possible -> is trivially possible
easy to use -> easy-to-use